src/frontend/src/components/SelectLanguageButtons.tsx

```diff
@@ -0,0 +1,40 @@
+import { useEffect, useState } from "react";
+
+import {
+    RadioButton,
+    RadioButtonGroup,
+} from "./base/RadioButtonGroup/RadioButtonGroup";
+
+export const Languages = {
```

Comment:
I stopped using enums in this project because: https://devblogs.microsoft.com/typescript/announcing-typescript-5-8-beta/#the---erasablesyntaxonly-option and instead followed the solution recommended here: https://dev.to/ivanzm123/dont-use-enums-in-typescript-they-are-very-dangerous-57bh